#Created by Quincy Fleming
#CFLAGS and LDFLAGS were not being respected
#Bug links: https://bugs.gentoo.org/910480 https://bugs.gentoo.org/910481
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,8 @@
 OUT := qdl
 
-CFLAGS := -O2 -Wall -g `pkg-config --cflags libxml-2.0`
-LDFLAGS := `pkg-config --libs libxml-2.0 libudev`
-prefix := /usr/local
+CFLAGS += `${PKG_CONFIG} --cflags libxml-2.0`
+LDFLAGS += `${PKG_CONFIG} --libs libxml-2.0 libudev`
+prefix = ${EPREFIX}/usr
 
 SRCS := firehose.c qdl.c sahara.c util.c patch.c program.c ufs.c
 OBJS := $(SRCS:.c=.o)
